### DISCUSSION TITLE
FPV.py  problem

### DISCUSSION THREAD

**Discussion by coo122567** [bitcraze/discussions#481] [Q&A]
Hello everyone, 
I would like to ask that an error occurred when I executed fpv.py, as shown in the figure below, I had no problem executing it before, but a problem occurred recently and I cannot solve it!
![image](https://user-images.githubusercontent.com/118537645/208454952-664f0aec-aada-427f-aaa3-6c7ada515ac0.png)

**Comment by krichardsson** [comment]
Duplicate of #474
